components/support/migration/src/main/java/mozilla/components/support/migration/session/InMemorySessionStoreParser.kt

```diff
@@ -44,7 +44,12 @@ internal object InMemorySessionStoreParser {
 
             val entry = entries.getJSONObject(index - 1)
             val url = entry.getString("url")
-            val title = entry.optString("title").ifEmpty { url }
+            val title = if (entry.isNull("title")) {
+                url
+            } else {
+                entry.optString("title").ifEmpty { url }
+            }
```

Comment:
nit: Can we do the same as we did below? `parsedTitle?.ifEmpty { url } ?: url`

Reply:
`optString` unfortunately doesn't give us `null` values and will return an empty String if you ask it for a value that is `null`.

Reply:
Got it!